### ISSUE TITLE
Converting .t7 labels into other formats

### ISSUE THREAD

**Issue by azmathmoosa** [1adrianb/2D-and-3D-face-alignment#28]
I am trying to use this dataset on a windows machine.  After wrestling with torch7 for a few days, I failed to get it working in Windows.  My aim is to train a network from scratch in dlib c++ library.  I need to first convert the .t7 labels (e. image00002.t7) into a different format.  

Since, I couldn't install torch, I am trying to use this [torch deserialization tool](https://github.com/bshillingford/python-torchfile) for python. 

When I try to open the file using the following code, I get an empty array. 
```
>>> torchfile.load('0001.t7')
array([], dtype=float64)
>>> torchfile.load('0011.t7')
array([], dtype=float64)
>>> torchfile.load('0111.t7')
array([], dtype=float64)
>>> torchfile.load('0110.t7')
array([], dtype=float64)
```

 Can you tell me the exact format of labels in the t7 file.  Or is there a utility you can point me to?  For now, I need the labels in plain text.  I will later decide what format to use.

**Comment by 1adrianb** [comment]
Hi @azmathmoosa , 

Are you using Windows? If yes, then there seems to be an issue with the library you are trying to use, see https://github.com/bshillingford/python-torchfile/issues/9

Regarding an utility to do this, you can use pytorch (there is a windows build for it using conda). If this still doesn't work, please let me know and I will have a look.

**Comment by azmathmoosa** [comment]
Thanks for the quick response @1adrianb .  I tried pytorch for windows and it too failed to deserialize.

```
>>> import torch
>>> from torch.utils.serialization import load_lua
>>>
>>> a = load_lua('aflw__face_40662.t7')
>>> a
[torch.DoubleTensor with no dimension]
```

Going through the issue that you referenced they say that binary t7 files are not portable due to different sizes of long ints. So I must use a linux system to deserialize them first and then switch to windows.  I will do that then.

Thanks for the assistance, and great work!

**Comment by SunlifeV** [comment]
I have finished the conversion, Win10, python3.7, pytorch 0.4.1(cpu)
use code 

```
import torch
from torch.utils.serialization import load_lua
# code...
t7_data = load_lua("the path", long_size=8)
# code...
```
